### DISCUSSION TITLE
dashboard-ReadLineLength.css not working with minimal theme

### DISCUSSION THREAD

**Discussion by AlvaroDiazBarrios** [TfTHacker/DashboardPlusPlus#20] [Q&A]
Hello everyone!

So my problem is that when i'm using minimal theme the _dashboard-ReadLineLength.css_ stops to work. Does anybody know how to fix this issue? I really like minimal theme but i would want to have the full with in my dashboard.

Thanks in advance!

**Comment by LYTWilson** [comment]
I've got the same problem as well.

**Comment by TfTHacker** [comment]
Thank you for reporting this. I have now updated dashboard-ReadLineLength.css  to work correctly with the minimal theme.
